src/main/java/gov/nasa/jpl/mbee/mdk/ems/validation/ElementValidator.java

```diff
@@ -142,7 +142,7 @@ else if (serverElement == null) {
                 validationRuleViolation.addAction(copyActionsCategory);
                 copyActionsCategory.addAction(new ClipboardAction("ID", id));
                 if (clientElementElement != null) {
-                    copyActionsCategory.addAction(new ClipboardAction("Element Hyperlink", "mdel://" + clientElementElement.getID()));
+                    copyActionsCategory.addAction(new ClipboardAction("Element Hyperlink", "mdel://" + Converters.getElementToIdConverter().apply(clientElementElement)));
```

Comment:
This is a case where we actually want to keep getID, because we're generating a hyperlink intended to be used with MagicDraw. Unless we can get rid of all our custom behavior in the converter, it may generate a hyperlink that MagicDraw can't identify.